setup.py

```diff
@@ -39,7 +39,7 @@ def find_version(*file_paths):
     'docker[ssh] >= 3.7.0, < 5',
     'dockerpty >= 0.4.1, < 1',
     'six >= 1.3.0, < 2',
-    'jsonschema >= 2.5.1, < 3',
+    'jsonschema >= 3, < 4',
```

Comment:
Please let the inferior bound as it is, since it still works with versions `>= 2.5.1`

Reply:
Seems reasonable.